### PR TITLE
OEL-1677: Add location facet to events.

### PR DESCRIPTION
**Pull request by drishu**
### Description

Add location facet to events.

### Change log

- Added:
  - patch to fix country labels
  - patch to fix error in facet
  - location facet
  - enabled oe_list_pages_address
  - oe_sc_event_location to index
- Fixed: whitelabel package require

### CODE REVIEW

File: modules/oe_showcase_list_pages/README.md [hunk 1]
```diff
@@ -30,3 +30,20 @@ Filters are facets on search API module. To create new facets you should:
...
+    "https://www.drupal.org/project/facets/issues/3262863": "https://git.drupalcode.org/project/facets/-/commit/88fd8a88a7206ea2e3828718fc404665ee20b064.diff"
+}
+```
+
+To use labels instead of country codes in address facets we added a patch:
+```composer log
+"openeuropa/oe_list_pages": {
+    "Support Address field in country code processor": "https://patch-diff.githubusercontent.com/raw/openeuropa/oe_list_pages/pull/139.diff"
+}
+```
```
Comment (donquixote) [inline modules/oe_showcase_list_pages/README.md:48]: Maybe this part can be refined, but let's do this in OEL-1516.

File: modules/oe_showcase_list_pages/README.md [hunk 2]
```diff
@@ -30,3 +30,20 @@ Filters are facets on search API module. To create new facets you should:
 * Click on "+Add facet".
 * Select the source and the fields added previously.
 * Click on save.
+
+### Note
+
+To overcome an [issue](https://www.drupal.org/project/facets/issues/3262863) with facets 2 we used below patch:
+
+```composer log
```
Comment (donquixote) [inline modules/oe_showcase_list_pages/README.md]: Is 'composer log' a language recognized by github or some other markdown viewer?

File: modules/oe_showcase_list_pages/README.md (hunk 1 above)
Comment (donquixote) [reply modules/oe_showcase_list_pages/README.md:48]: Blank lines are inconsistent..

**Review by donquixote — CHANGES_REQUESTED** [review]
Seems ok, but we need to enhance the test case.
I also left some notes in the readme, but we can also improve it later.

File: modules/oe_showcase_list_pages/README.md (hunk 2 above)
Comment (drishu) [reply modules/oe_showcase_list_pages/README.md]: github suggested, but I'll change to json, is better

File: modules/oe_showcase_list_pages/tests/src/ExistingSite/ListPagesTest.php [hunk 3]
```diff
@@ -223,6 +230,26 @@ public function testCreateListPages() {
     $title_input->setValue('News number 1');
     $search_button->click();
     $this->assertSearchResultsTitle('Event list page', 0);
+
+    // Filter results by location.
+    $filter_form->findButton('Clear filters')->click();
+    $location = $filter_form->findField('Location');
+    $location->selectOption('FR');
```
Comment (donquixote) [inline modules/oe_showcase_list_pages/tests/src/ExistingSite/ListPagesTest.php]: We should verify that country names are spelled out.
```suggestion
    $location->selectOption('France');
```
I tried this, it works.

**Review by escuriola — APPROVED** [review]

**Review by donquixote — APPROVED** [review]

**Pull request merged**